### PR TITLE
Try OSX 13 again

### PR DESCRIPTION
**Pull request by dougbu**
- see #45749

### CODE REVIEW

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -27,9 +27,7 @@
       <!-- aspnetcore-ci[-official] (ci.yml) -->
       <!-- aspnetcore-quarantined-pr (quarantined-pr.yml) -->
       <ItemGroup>
-        <HelixAvailableTargetQueue Include="Ubuntu.2004.Amd64.Open" Platform="Linux" />
```
Comment (wtgodbe) [inline eng/targets/Helix.Common.props:30]: Why get rid of this + Win?

Comment (dougbu) [reply eng/targets/Helix.Common.props:30]: This is a draft PR intended _only_ to test whether everything runs smoothly on OSX 13. No need to run tests anywhere else.

**Comment by halter73** [comment]
Does this passing mean we're no longer blocked on macOS 13 agents being added to the helix queue?

**Comment by dougbu** [comment]
No, see latest comments in #45749. The dashboard should be updated tomorrow to include the queue. Then, we watch the agent counts.

**Comment by Tratcher** [comment]
*ping*

**Comment by captainsafia** [comment]
Bump: have macOS 13 agents been added to the pool?

**Comment by dougbu** [comment]
> Bump: have macOS 13 agents been added to the pool?

Last we checked (in FR channel), the count was still at 10 and that's not sufficient for us to use on any kind of regular basis.

In addition, the Core-Eng Power BI dashboard still doesn't show the relevant queues 😦

**Comment by dougbu** [comment]
Another week has passed w/o https://msit.powerbi.com/groups/de8c4cb8-b06d-4af8-8609-3182bb4bdc7c/reports/8b49a89c-7aea-4fdb-95d5-547a9fe5d734/ReportSectione6503c9317531cbfd377?experience=power-bi changng to report anything about the new OSX 13 queues 😦

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**